Src/UIComponents/WeeklyGoalComponent/WeeklyGoalComponent.css

```diff
@@ -0,0 +1,22 @@
+.paragraph-week
+{
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 1.5rem;
+    text-align: center;
+}
```

Comment:
there seems to be unnecessary line break between week and {

Reply:
done